### ISSUE TITLE
Add details to README

### ISSUE THREAD

**Issue by saimatsumoto** [ProvidenceGeeks/website-backlog#4]
## Type of Change
- [ ] Improvement / Suggestion
- [ ] Other (please clarify below)

## Summary
Backlog can be utilized for:
1. **Planning:** backlog items are to be planned and estimated at a more granular level for the prospective sprints. 
2. **Prioritization:** enables us to map out what the next focus should be (i.e. the next major release) and to de-prioritize other issues (improvement, suggestion, low-priority bug). 
3. **Organization:** utilize backlog as a place for ideas and suggestions in order to avoid cluttering the ongoing project boards. 

## Other:
Please comment in for suggestions on ways to provide other needed information, if any.

**Comment by saimatsumoto** [comment]
@thescientist13 @kpisano I will appreciate any feedback!

**Comment by thescientist13** [comment]
Great idea @saimatsumoto !  I think using labels within this repo can help quickly apply context to issues based on what you've proposed here.

My only thought is for this repo (at least to start) is to only be a place for proposing NEW features / improvements.  A bug report for something in the frontend would start in that repo, for example.  If it turns out to require extensive work, then I could see that turning into an issue here.

In other words, I see this as being a more "product owner" oriented board.
